packages/features/src/categories/error-tracking/patterns/rollbar.ts

```diff
@@ -0,0 +1,128 @@
+import { Page } from 'playwright';
+
+export const rollbar = [
+  {
+    name: 'globals' as const,
+    score: 1.5,
+    browser: async (page: Page) => {
+      return page.evaluate(() => {
+        const markers = {
+          hasRollbar: !!window.Rollbar,
+          hasRollbarConfig: !!window._rollbarConfig,
+          hasRollbarAccessToken:
+            typeof window._rollbarAccessToken !== 'undefined',
+          hasRollbarShims: !!window._rollbarShims,
+          hasRollbarInitialized: !!window._rollbarInitialized,
+          hasRollbarWrappedError: !!window._rollbarWrappedError,
+          hasRollbarDidLoad: typeof window._rollbarDidLoad !== 'undefined',
+          hasRollbarOnError:
+            window.onerror &&
+            window.onerror.toString().includes('_rollbarWrappedError'),
+        };
+        return Object.values(markers).some(Boolean);
+      });
+    },
+  },
+  {
+    name: 'elements' as const,
+    score: 1.2,
+    browser: async (page: Page) => {
+      return page.evaluate(() => {
+        const markers = {
+          hasRollbarElements: !!document.querySelector('[data-rollbar]'),
+          hasRollbarScripts: Array.from(
+            document.querySelectorAll('script')
+          ).some((script) => {
+            if (!script.src && script.textContent) {
+              return (
+                script.textContent.includes('_rollbarConfig') ||
```

Comment:
This part is already checked via scripts (next section). In fact we do the same when we define scripts field